_rules/inline-link-in-paragraph-has-distinguishable-style-on-hover-and-focus-548868.md

```diff
@@ -0,0 +1,317 @@
+---
+id: 548868
+name: Inline link has distinguishing style or content on hover and focus
+rule_type: atomic
+description: |
+  This rule checks that inline links are distinguishable from the surrounding text through a difference in style or content when the links are hovered or focused.
+accessibility_requirements:
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Brian Bors
+    - Carlos Duarte
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any [visible][] HTML element that is a [semantic link][], for which all the following is true:
+
+- **link text**: the element has [visible][] [text nodes][text node] as [descendants][descendant] in the [flat tree][]; and
+- **non-link line text**: the element is [rendered on a line][] containing [visible][] [text nodes][text node] that are not [descendants][descendant] in the [flat tree][] of a [semantic link][]; or
+- **different color**: the element's [foreground color][] and the [foreground color][] of the **non-link line text** elements have a [highest possible contrast][] that is at least 3.0:1, or the element's [background color][] and the [background color][] of the **non-link line text** elements have a [highest possible contrast][] that is at least 3.0:1.
+
+## Expectation
+
+For each test target, at least one of the following is true when it is [focused][] or [hovered][]:
+
+- **distinguishing style**: the element has a [distinguishing style][] not based on color alone from the **non-link line text** elements; or
+- **distinguishing content**: element has content (such as an image or text), located in the proximity of the element (i.e. there is no other content visually in between this content and the target element), that indicates the test target is a link.
```

Comment:
Content can be anything, and I don't think we should go down the rabbit hole of trying to identify every possible content that can be used to "indicate the test target is a link". If "indicate the test target is a link" is not ambiguous I believe we're good. Any content that "indicates the test target is a link" would pass the rule. I don't think random white pixels, or images of black holes indicate a link. (But in certain contexts they might, and the rule would pass in those contexts).

Reply:
Agreed that it might end up too complex…
I'm a bit afraid that "content" could be a bit ambiguous. I guess "content that indicates the target is a link" should be good enough OTOH (subjective, but OK).
Maybe reordering the sentence around that could help 🤔 
"the element has content that indicates it is a link located in its proximity"
Hum, doesn't read super well either (does located refers to "link" or to "content that…"?)